### PR TITLE
Consider adding Layla, an android app that allows loading any GGUF models on Android phones to the list of supported UIs?

### PR DESCRIPTION
**Pull request by l3utterfly**
Layla allows you to load any GGUF model on Android phones for free.

Users can test GGUF capabilities by chatting with various preset and downloadable characters. Works offline, does not collect any data from conversations.

Would you consider adding this to the list of UIs that support llama.cpp?

Google Play link: https://play.google.com/store/apps/details?id=com.laylalite

### CODE REVIEW

**Comment by Jeximo** [comment]
@l3utterfly  As a user, I appreciate your work.  I want to point out that [in Description under UI](https://github.com/ggerganov/llama.cpp#description) it states, "*to have a project listed here, it should clearly state that it depends on llama.cpp*".  I may have missed the information.

**Comment by l3utterfly** [comment]
Oh ok. I will add that info the settings page and the play store description.

File: README.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ Unless otherwise noted these projects are open-source with permissive licensing:
 - [nat/openplayground](https://github.com/nat/openplayground)
 - [Faraday](https://faraday.dev/) (proprietary)
 - [LMStudio](https://lmstudio.ai/) (proprietary)
+- [Layla] (https://play.google.com/store/apps/details?id=com.laylalite) (proprietary)
```
Comment (compilade) [inline README.md:162]: There's an extra space between `[Layla]` and the link which prevents this from rendering correctly

Comment (l3utterfly) [reply README.md]: Fixed!

**Review by ggerganov — APPROVED** [review]

**Pull request merged**